app/views/media/_destination_cards.html.erb

```diff
@@ -125,7 +125,7 @@
                             </div>
                           </div>
                         <div class="col-2 mt-2 text-center">
-                          <div class="form-check custom-switch">
+                          <div class="form-check form-switch">
```

Comment:
Not necessarily related to this PR, however this is such a small change and goes along with replacing "custom-switch" with "form-switch"

Reply:
Fine with me.